tests/test_models.py

```diff
@@ -426,7 +426,22 @@ def test_get_remote_id(self, provider_id, expected_value, called, mock_third_par
 
     @mock.patch('enterprise.utils.segment')
     @mock.patch('enterprise.models.EnrollmentApiClient')
-    def test_enroll_learner(self, enrollment_api_client_mock, analytics_mock, *args):  # pylint: disable=unused-argument
+    @mock.patch('enterprise.models.EnterpriseCustomerUser.create_order_for_enrollment')
+    def test_enroll_learner(self, enrollment_order_mock, enrollment_api_client_mock, analytics_mock):
+        """
+        ``enroll_learner`` enrolls the learner and redirects to the LMS courseware.
+        """
+        enterprise_customer_user = factories.EnterpriseCustomerUserFactory()
+        enrollment_api_client_mock.return_value.get_course_enrollment.return_value = None
+        enterprise_customer_user.enroll('course-v1:edX+DemoX+Demo_Course', 'verified')
+        enrollment_api_client_mock.return_value.enroll_user_in_course.assert_called_once()
+        analytics_mock.track.assert_called_once()
+        enrollment_order_mock.assert_called_once()
+
+    @mock.patch('enterprise.utils.segment')
+    @mock.patch('enterprise.models.EnrollmentApiClient')
+    @mock.patch('enterprise.models.EnterpriseCustomerUser.create_order_for_enrollment')
+    def test_audit_enroll_learner(self, enrollment_order_mock, enrollment_api_client_mock, analytics_mock):
```

Comment:
I think we can merge this and the above test into a single test because they are same.

Reply:
Done